tests/engines/registries/unit/services/search-test.ts

```diff
@@ -20,35 +20,48 @@ class TestSearchFilter extends SearchFilter {
     }
 }
 
-module('Unit | Registries | Service | Search', hooks => {
+module('Unit | Service | search', hooks => {
```

Comment:
Seems like we should leave `Registries` in the test name somewhere?
Maybe `Registries | Unit | Service | search`?

Reply:
👍